frappe/public/js/frappe/ui/filters/filter_list.js

```diff
@@ -130,7 +130,7 @@ frappe.ui.FilterGroup = class {
 	update_filter_button() {
 		const filters_applied = this.filters.length > 0;
 		const button_label = filters_applied
-			? __("Filters <span class='filter-label'>{0}</span>", [this.filters.length])
+			? __("Filters") + `<span class='filter-label'>${this.filters.length}</span>`
```

Comment:
```suggestion
			? __("Filters {0}", [`<span class="filter-label">${this.filters.length}</span>`])
```

This way, translators will know that a variable is following and the whitespace is preserved.

Reply:
Btw, your PR title is misleading, because this string is already translatable and translated ;)

Reply:
Thanks!